tests/objectstore/s3ssekms/TestS3SseKms.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Test S3 ObjectStorage with SSE KMS encryption
+ *
+ * @author Bernd Rederlechner
+ * @copyright 2021 T-Systems International (bernd.rederlechner@t-systems.com)
+ * @license AGPL-3.0
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+namespace Test;
+
+use OC\Files\ObjectStore\S3;
+use PHPUnit\Framework\TestCase;
+
+
+/**
+ * Run this acceptance test against a real S3 object store
+ * (it is difficult to mock out S3)
+ * 
+ * The test command is:
+ * > NC_S3_HOST=... NC_S3_REGION=... NC_S3_AK=... NC_S3_SK=... NC_S3_Bucket NC_S3_KMS_ID=...  
+ */
+class TestWithSseKms extends TestCase {
+	protected $config = [
+		'objectstore' => [
+			'class' => '\\OC\\Files\\ObjectStore\\S3',
+			'arguments' => [
+				'autocreate' => true,
+				'use_path_style'=>true
+			],
+		], 
+		'installed' => true
+	];
+	protected $client;
+
+    /**
+     * Create an initial object in existing bucket
+     */
+	protected function setUp(): void {
+		$this->config['objectstore']['arguments']['bucket'] = getenv("NC_S3_BUCKET");
+		$this->config['objectstore']['arguments']['key'] = getenv("NC_S3_AK");
+		$this->config['objectstore']['arguments']['secret'] = getenv("NC_S3_SK");
+		$this->config['objectstore']['arguments']['hostname'] = getenv("NC_S3_HOST") ?: 'obs.eu-de.otc.t-systems.com';
```

Comment:
Let's remove the hard-coded link :)

```suggestion
		$this->config['objectstore']['arguments']['hostname'] = getenv("NC_S3_HOST");
```

Reply:
Let´s remove this experimental `TestS3SseKms.php` file completely. I made this before I found the correct Unittests for it. I thought that I took it out of the merge, but now it came back...
Sorry